src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -7183,6 +7701,54 @@ class C
             edits.VerifySemanticDiagnostics();
         }
 
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Event_ExpressionBodyToBlockBody()
+        {
+            string src1 = @"
+using System;
+public class C
+{
+    event Action E { add => F(); remove => F(); }
+}
+";
+            string src2 = @"
+using System;
+public class C
+{
+   event Action E { add { F(); } remove => F(); }
```

Comment:
>=> F();  [](start = 40, length = 8)

Perhaps change this also to { }, so that we cover remove handler changing from block to expression body as well.